R/disambiguate.R

```diff
@@ -25,7 +25,7 @@
 #' dm_nycflights13() %>%
 #'   dm_disambiguate_cols()
 #' @export
-dm_disambiguate_cols <- function(dm, sep = ".", ..., quiet = FALSE) {
+dm_disambiguate_cols <- function(dm, sep = ".", ..., .quiet = FALSE) {
```

Comment:
Are these empty dots ? If so should we dot the ".quiet" ? if so shouldn't we dot the "sep" ?

Reply:
The dots are empty for now but might get semantics later -- the tables we want to disambig columns for as a tidyselect. Good point about dotting `.sep` .